### PR TITLE
[16.0][Feat] Added store location type contact

### PR DESCRIPTION
**Pull request by wouitmil**
Screenshot 2024-07-04 at 11 20 13

### CODE REVIEW

File: partner_store/models/res_partner.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import logging
```
Comment (yvaucher) [inline partner_store/models/res_partner.py:3]: Missing copyright

**Review by yvaucher — CHANGES_REQUESTED** [review]
Copyright is missing. Please fix this.

File: partner_store/models/res_partner.py (hunk 1 above)
Comment (wouitmil) [reply partner_store/models/res_partner.py:3]: Done

**Review by yvaucher — APPROVED** [review]

**Review by StephaneMangin — APPROVED** [review]
Nice !

File: partner_store/README.rst [hunk 2]
```diff
@@ -0,0 +1,64 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```
Comment (gurneyalex) [inline partner_store/README.rst]: This file is supposed to e generated from documents in the readme/ subdirectory but this directory is missing. 

Additionally the content of the file is suspicious. I see no "geospatial support" in the module, for instance.

Comment (wouitmil) [reply partner_store/README.rst]: Better like this?

**Pull request merged**